neuralmonkey/config/parsing.py

```diff
@@ -130,10 +130,12 @@ def _parse_list(string: str, vars_dict: VarsDict) -> List[Any]:
 
     items = _split_on_commas(matched_content)
     values = [_parse_value(val, vars_dict) for val in items]
-    types = [type(val) for val in values]
 
-    if len(set(types)) > 1:
-        raise ParseError("List must of a same type, is: {}".format(types))
+    # Do not check types because the dataset specs list are not homogeneous
+    # types = [type(val) for val in values]
+
+    # if len(set(types)) > 1:
+    #     raise ParseError("List must of a same type, is: {}".format(types))
```

Comment:
Proč to celý nesmažeš?

Reply:
Aby sis toho všimnul při review a kdyžtak řek že to nechcem rušit.

Reply:
Od tý doby, co jedeme typeguardy, tak už to není potřeba, ne?

Reply:
řek bych že ne; smažu to